setup.cfg

```diff
@@ -17,13 +17,15 @@ package_dir =
     = src
 packages = find:
 python_requires = >=3.8
+install_requires =
+    pip-tools
 
 [options.packages.find]
 where = src
 
 [options.entry_points]
 console_scripts =
-    pip-sync-faster = pip_sync_faster.main:entry_point
+    pip-sync-faster = pip_sync_faster.cli:cli
```

Comment:
I'll update this in the cookiecutter: `main.py::entry_point()` renamed to `cli.py::cli()`